libraries/standard/http/include/http_client.h

```diff
@@ -207,32 +207,44 @@ typedef struct HTTPRequestInfo
     uint32_t flags;
 } HTTPRequestInfo_t;
 
+
+
 /**
  * @brief Callback to intercept headers during the first parse through of the 
  * response as it is received from the network.
  */
 typedef struct httpResponseParsingCallback {
     /**
-     * Invoked when both a header field and its associated header value is found.
+     * @brief Invoked when both a header field and its associated header value are found.
      * @param[in] pContext User context.
-     * @param[in] fieldLoc Pointer to the first char of the header field name in the response buffer.
+     * @param[in] fieldLoc Location of the header field name in the response buffer.
      * @param[in] fieldLen Length in bytes of the field name.
-     * @param[in] valueLoc Pointer to the first char of the header value in the response buffer.
+     * @param[in] valueLoc Location of the header value in the response buffer.
      * @param[in] valueLen Length in bytes of the value.
      * @param[in] statusCode The HTTP response status-code.
      */
     void (*onHeaderCallback)( void* pContext, const char* fieldLoc, size_t fieldLen, const char* valueLoc, size_t valueLen, uint16_t statusCode );
 
     /* Private context for the application. */
     void *pContext;
-} httpclientHeaderParsingCallback_t;
+} HTTPClient_HeaderParsingCallback_t;
 
+/**
+ * @brief Represents an HTTP response.
+ */
 typedef struct HTTPResponse
 {
     /**
      * @brief Buffer for both the raw HTTP header and body. 
      * 
-     * This buffer cannot be modifed until #HTTPClient_Send returns.
+     * This buffer is supplied by the application.
+     * 
+     * This buffer is owned by the library during  #HTTPClient_Send. This buffer
```

Comment:
This would also be owned by `HTTPClient_ReadHeader` wouldn't it?

Reply:
Yes forgot will add.